File: src/codes/Users_Manual/link/simulator_0.cs
~using AUTD3Sharp;
using AUTD3Sharp.Link;

~using AUTD3Sharp.Gain;
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Utils;
~var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(
Simulator.Builder(8080)
~);

File: src/codes/Users_Manual/link/simulator_0.rs
# use autd3::prelude::*;
use autd3_link_simulator::Simulator;

# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let autd = Controller::builder([AUTD3::new(Vector3::zeros())])
#             .open(
Simulator::builder(8080)
# ).await?;
# Ok(())
# }